rasa_nlu/model.py

```diff
@@ -226,7 +227,7 @@ def __init__(self, pipeline, context, config, meta=None):
         # type: (List[Component], Dict[Text, Any], Dict[Text, Any], Optional[Metadata]) -> None
 
         self.pipeline = pipeline
-        self.context = context
+        self.context = self.default_output_attributes.update(context)
```

Comment:
Does this work? wont the call a) update the default output attributes and b) return nothing?